libafl/src/feedbacks/list.rs

```diff
@@ -0,0 +1,159 @@
+use alloc::string::{String, ToString};
+use core::{fmt::Debug, hash::Hash, marker::PhantomData};
+
+use hashbrown::HashSet;
+use libafl_bolts::{Error, Named};
+use serde::{de::DeserializeOwned, Deserialize, Serialize};
+
+use crate::{
+    events::EventFirer,
+    executors::ExitKind,
+    feedbacks::Feedback,
+    observers::{ListObserver, ObserversTuple},
+    state::{HasNamedMetadata, State},
+};
+
+/// The prefix for list metadata's name
+pub const LISTFEEDBACK_PREFIX: &str = "listfeedback_metadata_";
```

Comment:
Don't think we need a prefix since it's typed anyway

Reply:
I followed what mapfeedbackmetadata did.
so for mapfeedbackmetadata the prefix is not required too?

Reply:
I don't think so (and it would probably make lookups a tiny bit faster, lol)

Reply:
ok. i didn't get the purpose of this prefix either.